src/_posts/security/procedures/2000-01-01-load-test.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Running a load test
+nav: Load test
+modified_at: 2021-10-08 00:00:00
+tags: security pentest procedures
+---
+
+## Can I run a load test on my application that is running on Scalingo?
```

Comment:
It's quite a bad practice for us to have plenty of really small pages.

Could you maybe gather the procedures into some kind of Q&A?

Reply:
> It's quite a bad practice for us to have plenty of really small pages.

Agree with Leo on this one

> Could you maybe gather the procedures into some kind of Q&A?

As we don't want to use lot of time and we want this documentation soon, I think Q&A can be a good tool.
But be aware that most of the times Q&A aren't good documentation https://idratherbewriting.com/2017/06/23/why-tech-writers-hate-faqs/